### DISCUSSION TITLE
Does Arms warrior have AOE cycle? I didn't find it

### DISCUSSION THREAD

**Discussion by GonzoXing** [simulationcraft/simc#6167] [Q&A] [answered]
I don't know where to look

**Comment by vituscze** [comment, marked as answer]
I believe it's this part of their APL: https://github.com/simulationcraft/simc/blob/6f3d3dd3756ab1e8386d7ff935982c7cff623ad3/profiles/Tier27/T27_Warrior_Arms.simc#L67-L81

**Reply by GonzoXing** [reply]
thx，dude！
